### DISCUSSION TITLE
[FEATURE]: Add ShopAbout ShopAboutMembers to API V3

### DISCUSSION THREAD

**Discussion by gerlv** [etsy/open-api#711] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

There is no way to get `about us` story or shop members via API, and it's a requirement for our application.

### Describe the solution you'd like

An endpoint that returns shop About and About Members sections

### Describe any alternatives you've considered

None, it's not available in API v3, only in API v2, but it's going away, and new applications can't even access API v2 anymore.

### Additional context

There is a [similar ticket](https://github.com/etsy/open-api/issues/595), but it's about preserving v2 resources, ideally, this should be implemented in API v3.

**Comment by etsyachristensen** [comment]
@gerlv Your request has been logged for discussion.

**Comment by etsyachristensen** [comment]
We are closing the issues board in favor of the Discussions system. We'll convert this to a new discussion in the Ideas category.
